Simplify legacy close event mapping

Abban opened the pull request:
Legacy close event action names will now be created using the values in the new close event object.

timEulitz commented on english/templates/banner_text.hbs:
```diff
@@ -17,5 +17,9 @@
+    <span class="ctrl-text">Please take a minute to help us keep Wikipedia growing.</span>
+    <span class="var-text">If Wikipedia gave you <span class="no-wrap">€ 5</span> worth of knowledge this year, take a minute to donate and help us to keep Wikipedia growing.</span>
+    Thank you!
+    <span class="ctrl-text--invisible">______________I'm an invisible placeholder____________________</span>
```
It's better to re-use the exact same text again to be sure the banner will break at the same time. Usually we create a banner_text_var for this but if you want to stick to one _text.hbs that's fine. In that case it would look like that:

```
<span class="ctrl-text--invisible">If Wikipedia gave you <span class="no-wrap">€ 5</span> worth of knowledge this year, take a minute to donate and help us to keep Wikipedia growing.</span>
<span class="var-text--invisible">Please take a minute to help us keep Wikipedia growing.</span>
```

And then ctrl-text-invisible would be "hidden" on VAR and var-text--invisible would be 'display:none' on VAR (and vice versa on CTRL).

moiikana replied:
maybe I thought too complicated about that again, but I needed to keep the banners the exact same height and tried to achieve it with this invisible text. The --invisible span contains the exact amount of characters that is needed to have the same total amount of characters in CTRL and VAR (it felt very horrible doing this but it seemed to be the quickest and most effective way to achieve equal height here without having to add lots of extra space for both banners and making them unnecessarily big).
_length(ctrl-text--invisible) = length(var-text) - length(ctrl-text)_

timEulitz replied:
Hey, the problem with that is that most fonts are not "monospaced" which means different characters have different lengths, especially across browsers / operating systems. "W" and "_" are generally wider characters than "i" or "t". That's why the only way to really get it right is to make an exact copy of the text. I personally don't care that much but Till and the gang had complaints about this in the past.

Abban committed:
Simplify legacy close event mapping

Legacy close event action names will now be created using the
values in the new close event object.

gbirke reviewed (approved):
If the PM is fine with this, I am too, this makes the banner smaller and the code easier.
